README.md

```diff
@@ -105,6 +105,31 @@ Refer to the [documentation](https://docs.microsoft.com/dotnet/api/microsoft.sql
 
 **Note:** If you are replacing an existing `.sqlproj` be sure to copy over any of these properties into the new project file.
 
+## Model compiler options
+Like `.sqlproj` projects  `MSBuild.Sdk.SqlProj` supports special `msbuild` options to control T-SQL build errors and warning.
+Treating warnings as errors can be optionally enabled by adding a property `TreatTSqlWarningsAsErrors` to the project file:
+
+```xml
+<Project Sdk="MSBuild.Sdk.SqlProj/1.11.0">
```

Comment:
We'll be releasing this as a new minor version since this adds a new feature. I suggest doing a find replace for `MSBuild.Sdk.SqlProj/1.11.0` with `MSBuild.Sdk.SqlProj/1.12.0`.

Reply:
Looks like this is fixed only for the first occurrence. Ideally we'd have all the code samples reference the latest version.